src/deprecated.jl

```diff
@@ -282,3 +282,70 @@ function insertcols!(df::DataFrame, col_ind::Int; makeunique::Bool=false, name_c
     length(name_col) == 1 || throw(ArgumentError("one and only one column must be provided"))
     return insertcols!(df, col_ind, makeunique=makeunique, keys(name_col)[1] => name_col[1])
 end
+
+function Base.join(df1::AbstractDataFrame, df2::AbstractDataFrame;
+                   on::Union{<:OnType, AbstractVector} = Symbol[],
+                   kind::Symbol = :inner, makeunique::Bool=false,
+                   indicator::Union{Nothing, Symbol} = nothing,
+                   validate::Union{Pair{Bool, Bool}, Tuple{Bool, Bool}}=(false, false))
+    if kind == :inner
+        Base.depwarn("inner joining data frames using join is deprecated, use " *
```

Comment:
Why not show the exact syntax (even if it's verbose)?

Reply:
OK - added